include/seqan3/alphabet/composition/cartesian_composition.hpp

```diff
@@ -136,24 +315,203 @@ struct cartesian_composition :
     }
     //!\}
 
-private:
-    //!\brief declared private to prevent direct use of the CRTP base
-    cartesian_composition() = default;
-    //!\brief declared private to prevent direct use of the CRTP base
-    constexpr cartesian_composition(cartesian_composition const &) = default;
-    //!\brief declared private to prevent direct use of the CRTP base
-    constexpr cartesian_composition(cartesian_composition &&) = default;
-    //!\brief declared private to prevent direct use of the CRTP base
-    constexpr cartesian_composition & operator =(cartesian_composition const &) = default;
-    //!\brief declared private to prevent direct use of the CRTP base
-    constexpr cartesian_composition & operator =(cartesian_composition &&) = default;
-    //!\brief declared private to prevent direct use of the CRTP base
-    ~cartesian_composition() = default;
-
     //!\brief befriend the derived type so that it can instantiate
     //!\sa https://isocpp.org/blog/2017/04/quick-q-prevent-user-from-derive-from-incorrect-crtp-base
     friend derived_type;
 
+    // Inherit default comparators explicitly from base class
+    using base_t::operator==;
+    using base_t::operator!=;
+    using base_t::operator<;
+    using base_t::operator>;
+    using base_t::operator<=;
+    using base_t::operator>=;
+
+    /*!\name Member comparison operators with one of its base alphabet type or subtype.
+     * \brief This enables the comparison of a cartesian composition value against
+     * a value of the inner **alphabet type** while ignoring the other values.
+     * The operators delegate to the comparison operators of the alphabet type.
+     * \{
+     */
+    //!\brief Returns `true` if \p std::get<alphabet_t>(*this) == \p rhs, `false` otherwise.
```

Comment:
I don't think these docs really help.
I think you also do not need them, because all the comparison operators are contained in the `\name Member comparison...` group?